generator/src/lib.rs

```diff
@@ -38,33 +42,58 @@ impl quote::ToTokens for CType {
 
 named!(ctype<&str, CType>,
     alt!(
-        tag!("ULL") => { |_| CType::U64 } |
-        tag!("U") => { |_| CType::U32  }
+        value!(CType::U64, complete!(tag!("ULL"))) |
+        value!(CType::U32, complete!(tag!("U")))
```

Comment:
This was never working before: the longest prefix needs to be matched first but at the same time that sets the minimum length requirement for the rest of the `alt!`, so `tag!("U")` never got matched.

Documented here, perhaps this should be a comment in-line:
https://github.com/Geal/nom/blob/38bb94e7bc45c2b4ef0c473e0e5c03ab134cdff3/src/branch/macros.rs#L93-L142